app/Http/Controllers/Api/ReactionController.php

```diff
@@ -61,7 +63,15 @@ public function store(ReactionRequest $request)
             }
         }
 
-        $totalReactions = $this->getTotalPostReactions($postableId, $postableType);
+        $reactionableType = explode('\\', $reactionableType)[2];
```

Comment:
Is this to grab the non-namespaced class name? I'm a little worried what happens if a well-formatted string isn't passed here... e.g. if an application accidentally passed `reactionable_type => 'photo'`.

Other applications shouldn't have to know about the internal Rogue class names, so it might make sense to have a mapping of `'photo' => \App\Models\Photo::class` here to allow the API to just use always "short" names & transform them to the appropriate namespaced class in the controller or service class.